src/stdatm/atmosphere.py

```diff
@@ -174,31 +193,25 @@ def kinematic_viscosity(self) -> Union[float, np.ndarray]:
             self._kinematic_viscosity = compute_kinematic_viscosity(self.temperature, self.density)
         return self._kinematic_viscosity
 
-    @property
-    def mach(self) -> Union[float, np.ndarray]:
-        """Mach number."""
-        if self._mach is None and self.true_airspeed is not None:
-            self._mach = self.true_airspeed / self.speed_of_sound
-        return self._mach
-
     @property
     def true_airspeed(self) -> Union[float, np.ndarray]:
         """True airspeed (TAS) in m/s."""
         # Dev note: true_airspeed is the "hub". Other speed values will be calculated
         # from this true_airspeed.
         if self._true_airspeed is None:
             if self._mach is not None:
-                self._true_airspeed = self._mach * self.speed_of_sound
+                self._true_airspeed = compute_tas_from_mach(self._mach, self.speed_of_sound)
             elif self._equivalent_airspeed is not None:
-                self._true_airspeed = self._equivalent_airspeed * np.sqrt(
-                    SEA_LEVEL_ATMOSPHERE.density / self.density
+                self._true_airspeed = compute_tas_from_eas(
+                    self._equivalent_airspeed, self.density, SEA_LEVEL_ATMOSPHERE.density
                 )
             elif self._unitary_reynolds is not None:
-                self._true_airspeed = self._unitary_reynolds * self.kinematic_viscosity
-            elif self._dynamic_pressure is not None:
-                self._true_airspeed = (
-                    np.sqrt(self._dynamic_pressure / 0.7 / self.pressure) * self.speed_of_sound
+                self._true_airspeed = compute_tas_from_unit_re(
+                    self._unitary_reynolds, self.kinematic_viscosity
                 )
+            elif self._dynamic_pressure is not None:
+                self._mach = compute_mach_from_pdyn(self._dynamic_pressure, self.pressure)
+                self._true_airspeed = compute_tas_from_mach(self._mach, self.speed_of_sound)
```

Comment:
Is it necessary to use the mach number to compute the true airspeed ? Especially since `compute_tas_from_pdyn` exist
Also, if we keep that method, shouldn't we use self.mach and not self._mach ?

Reply:
Actually, I forgot to remove `compute_tas_from_pdyn`. I wrote it first, then I realized the mach number had to be an intermediate result, and that it was more efficient to store it already. In the case the class gets dynamic pressure as an input, is asked for TAS, and then is asked for Mach number, there would be a superfluous division to do.
Using `compute_tas_from_pdyn` would have the advantage of consistency (having all speed parameters converted to TAS first), but since we aim now at optimization, I think it's better to be pragmatic and avoid any useless operation. Do you agree?

And about self._mach, the operations done in the setter and getter of the property are already guaranteed here, so it is simply more efficient to address the actual attribute here. Do you feel like a comment would be needed here?

Reply:
Actually, no, it is not mandatory. I just wanted to stick to the same equation based on Mach and pressure for... some reason. But finally, I cannot see why we could not use the good old 0.5 x rho x V². I'll change that, it will get more consistent.

Reply:
Done in 7d2007d25f8ddc0626f7bd8931f54ed5d0ec843d.